akka-docs/src/main/paradox/stream/stream-parallelism.md

```diff
@@ -82,7 +82,7 @@ since that would require a third processing step, which is not practically possi
 
 One drawback of the example code above that it does not preserve the ordering of pancakes. This might be a problem
```

Comment:
While you are in here making a change, you could also consider
```suggestion
One drawback of the example code above is it does not preserve the ordering of pancakes. This might be a problem
```